atlp-rwanda/commanders-rn-medica
# Feature forgot password with phone number

What does this PR do?
This PR includes the implementation of the reset password for registered users, where an OTP is sent to their phone number so that it can be used for resetting the password.

How should this be manually tested?
Navigate to the sign-in page and  click on the forgotten password. You will be directed to a screen where you will choose the method you want to use to reset your password. From this screen, you will  choose the field of phone number to fill in the OTP, which is the one you received on your phone number a modal will pop up confirming that your password is set and then you will be redirected to the home screen.

### Steps to test it:
- Create an account if you don't have one
- Login and update the profile with your phone starting with 250
- You logout then head to reset password make sure the phone starts with 250

⚠️ NB: if the number isn't related to your account it'll not work!
